miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PrinterRestController.java

```diff
@@ -112,19 +115,19 @@ public void delete(@RequestBody List<Long> printerIds) throws IOException {
     }
   }
 
-  @RequestMapping(value = "/disable", method = RequestMethod.PUT, headers = { "Content-type=application/json" })
+  @PutMapping(value = "/disable", headers = { "Content-type=application/json" })
   @ResponseStatus(HttpStatus.NO_CONTENT)
   public void disable(@RequestBody List<Long> printerIds) throws IOException {
     setState(printerIds, false);
   }
 
-  @RequestMapping(value = "/enable", method = RequestMethod.PUT, headers = { "Content-type=application/json" })
+  @PutMapping(value = "/enable", headers = { "Content-type=application/json" })
   @ResponseStatus(HttpStatus.NO_CONTENT)
   public void enable(@RequestBody List<Long> printerIds) throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)